File: teste.py
def calculator():
    print("Calculadora Simples")
    print("Escolha a operação:")
    print("1. Adição")
    print("2. Subtração")
    print("3. Multiplicação")
    print("4. Divisão")
    
    escolha = input("Digite o número da operação desejada: ")
    
    if escolha in ('1', '2', '3', '4'):
        num1 = float(input("Digite o primeiro número: "))
        num2 = float(input("Digite o segundo número: "))
        
        if escolha == '1':
            print(f"Resultado: {num1} + {num2} = {num1 + num2}")
        elif escolha == '2':
            print(f"Resultado: {num1} - {num2} = {num1 - num2}")
        elif escolha == '3':
            print(f"Resultado: {num1} * {num2} = {num1 * num2}")
        elif escolha == '4':
            if num2 != 0:
                print(f"Resultado: {num1} / {num2} = {num1 / num2}")
            else:
                print("Erro: Divisão por zero não é permitida.")
    else:
        print("Opção inválida. Tente novamente.")

if __name__ == "__main__":
    calculator()

File: teste.go
package main

import (
    "fmt"
)

func main() {
    fmt.Println("Calculadora Simples")
    fmt.Println("Escolha a operação:")
    fmt.Println("1. Adição")
    fmt.Println("2. Subtração")
    fmt.Println("3. Multiplicação")
    fmt.Println("4. Divisão")

    var escolha int
    fmt.Print("Digite o número da operação desejada: ")
    fmt.Scan(&escolha)

    if escolha >= 1 && escolha <= 4 {
        var num1, num2 float64
        fmt.Print("Digite o primeiro número: ")
        fmt.Scan(&num1)
        fmt.Print("Digite o segundo número: ")
        fmt.Scan(&num2)

        switch escolha {
        case 1:
            fmt.Printf("Resultado: %.2f + %.2f = %.2f\n", num1, num2, num1+num2)
        case 2:
            fmt.Printf("Resultado: %.2f - %.2f = %.2f\n", num1, num2, num1-num2)
        case 3:
            fmt.Printf("Resultado: %.2f * %.2f = %.2f\n", num1, num2, num1*num2)
        case 4:
            if num2 != 0 {
                fmt.Printf("Resultado: %.2f / %.2f = %.2f\n", num1, num2, num1/num2)
            } else {
                fmt.Println("Erro: Divisão por zero não é permitida.")
            }
        }
    } else {
        fmt.Println("Opção inválida. Tente novamente.")
    }
}
